tests/unit/introspection.tcl

```diff
@@ -166,11 +166,26 @@ start_server {tags {"introspection"}} {
             assert_match [r config get save] {save {3600 1 300 100 60 10000}}
         }
 
-        # First "save" keyword overrides defaults
+        # First "save" keyword overrides hard coded defaults
         start_server {config "minimal.conf" overrides {save {100 100}}} {
             # Defaults
             assert_match [r config get save] {save {100 100}}
         }
+
+        # First "save" keyword in default config file
+        start_server {config "default.conf"} {
+            assert_match [r config get save] {save {900 1}}
+        }
+
+        # First "save" keyword appends default from config file
+        start_server {config "default.conf" args {--save 100 100}} {
+            assert_match [r config get save] {save {900 1 100 100}}
+        }
+
+        # Empty "save" keyword resets all
+        start_server {config "default.conf" args {--save {}}} {
+            assert_match [r config get save] {save {}}
+        }
```

Comment:
Maybe you'd like to add a test with a config file like this:
```
save 900 1
save ""
```
I think the code now should support resetting the `save` option from within the config (not only from cmd line).
Maybe this is an overkill.

Reply:
Yes, it'll work as well, but I don't have an infrastructure to generate config files with multiple lines of the same config.

Reply:
i've tested this manually, as well as just
```
save 900 1
save
```
both work (the empty save clears everything)